pages/1.10/release-notes/1.10.10/index.md

```diff
@@ -0,0 +1,186 @@
+
+
+## Breaking Changes
+
+- Marathon Networking API Changes in 1.5.
+
+  The networking section of the Marathon API has changed significantly in version 1.5. Marathon can still accept requests using the 1.4 version of the API, but it will always reply with the 1.5 version of the app definition. This will break tools that consume networking-related fields of the service definition. [View the documentation](https://github.com/mesosphere/marathon/blob/master/docs/docs/networking.md).
+
+- TLS 1.0 is no longer enabled by default in Admin Router. [enterprise type="inline" size="small" /]
+
+  TLS 1.0 no longer meets common minimum security requirements. To use TLS 1.0, set `adminrouter_tls_1_0_enabled` to `true` in your `config.yaml` at install time. The default is `false`.
+
+- Moved file location for the DC/OS CA bundle in the sandbox of Mesos tasks from `$MESOS_SANDBOX/.ssl/ca.crt` to `$MESOS_SANDBOX/.ssl/ca-bundle.crt` and declared the new file path to be stable.
+
+- Marathon-LB 1.11.0 or greater is required for DC/OS 1.10.
+
+  Before upgrading to DC/OS 1.10, uninstall your existing Marathon-LB package and reinstall the updated version.
+
+- REX-Ray configuration change.
+
+  DC/OS 1.10 upgrades REX-Ray from v0.3.3 to v0.9.0 and the REX-Ray configuration format has changed. If you have specified custom REX-Ray configuration in the [`rexray_config`](/1.10/installing/oss/custom/configuration/configuration-parameters/#rexray-config) parameter of your `config.yaml` file, either update the configuration to the new format or remove `rexray_config` and set the parameter to `rexray_config_preset: aws`, which configures the `rexray_config` parameter to the default REX-Ray configuration bundled with DC/OS. This option has the benefit of automatically upgrading your cluster's REX-Ray configuration when you upgrade to a newer version of DC/OS.
+  **Note:** The `rexray_config_preset: aws` option is only relevant to DC/OS clusters running on AWS.
+
+- New flow to change the `dcos_url` and log in.
+
+  The new command to set up your cluster URL is `dcos cluster setup <dcos_url>`. For details, see [CLI](#cli).
+
+- Hard CFS CPU limits enabled by default.
+
+  DC/OS 1.10 enforces hard CPU limits with CFS isolation for both the Docker and Universal Container Runtimes. This will give more predictable performance across all tasks but might lead to a slowdown for tasks (and thereby also deployments) who have previously have consumed more CPU cycles than allocated. See [MESOS-6134](https://issues.apache.org/jira/browse/MESOS-6134) for more details.
+
+## Known Issues
+- Upgrade: During upgrade to DC/OS 1.10, there is a brief moment when the DNS resolution does not work. If a health check runs at that moment, it will fail and services will be reported as unhealthy.
+- CORE-1125 - Docker image pull config is re-used.
+- DCOS-16547 - Task state does not update after the agent running it was removed from the cluster.
```

Comment:
@megatmar Another suspicious ticket...that doesn't belong.

Reply:
Do you mean all of the known issues (again a holdover from previous release notes).

Reply:
Hidden content.